### DISCUSSION TITLE
Add RTC support to FRDM-MCXN947

### DISCUSSION THREAD

**Discussion by iustmehri** [zephyrproject-rtos/zephyr#76443] [Q&A] [answered]
As far as I know zephyr does not support RTC for FRDM-MCXN947. Can you please add RTC support for this board?

**Comment by DerekSnell** [comment, marked as answer]
Moved to Enhancement Request (Issue) at https://github.com/zephyrproject-rtos/zephyr/issues/76487.  Closing this Discussion.
